lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2054 @@
+{
```

Comment:
<p>Asi neuškodí něco takového zopakovat, ale tohle by měly lidi na kurzu už umět, takže víc času je možné věnovat pandas a spol. Já bych klidně tyhle dva for cykly vynechal :)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>